src/main/java/frc/robot/vision/RawVisionPoseData.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public record RawVisionPoseData(Pose3d estimatedPose, double aprilTagHeight, double distanceFromAprilTag, double timestamp) {}
```

Comment:
```suggestion
public record RawVisionRobotPoseData(Pose3d estimatedPose, double aprilTagHeight, double distanceFromAprilTag, double timestamp) {}
```

Reply:
changed to RawVisionAprilTagData since it's not just the robot pose